### PR TITLE
Elasticsearch search as you type

### PR DESCRIPTION
**Pull request by sikachu**
This functionality relies on https://search.chapman.edu to become available.

Here's how it looks:

search

---

Currently, it's waiting for #16 to be merged.

### CODE REVIEW

File: src/omni-nav.js [hunk 1]
```diff
@@ -523,7 +359,34 @@ this.jQuery && (function ($) {
...
+					globalSearch.currentSearchTerm = jQuery.trim($(this).val());
+					var baseSearchUrl = $('#cu_search_box form').prop('action');
+
+					if(globalSearch.currentSearchTerm != '') {
+						var searchUrl = baseSearchUrl +
+							'?query=' +
+							encodeURIComponent($(this).val());
+
+						$('#cu_search_result .result_container').
+							load(searchUrl + ' #search-results .search-result', function() {
```
Comment (jferris) [inline src/omni-nav.js]: Huh, I hadn't seen this syntax for `load` before. Neat.

**Comment by jferris** [comment]
Maybe include a screenshot of the new search results in action given the CSS and markup changes/

**Comment by sikachu** [comment]
Updated with screenshot.

**Comment by tatwell** [comment]
Closing and deleting branch without merging.

**Pull request closed without merging**